sh/upload.sh

```diff
@@ -676,16 +724,22 @@ main() {
     _setup_arguments "${@}"
     "${SKIP_INTERNET_CHECK:-_check_internet}"
 
-    [ -n "${PARALLEL_UPLOAD}" ] && {
-        { command -v mktemp 1>| /dev/null && TMPFILE="$(mktemp -u)"; } || TMPFILE="$(pwd)/$(date +'%s').LOG"
-    }
+    { command -v mktemp 1>| /dev/null && TMPFILE="$(mktemp -u)"; } || TMPFILE="$(pwd)/.$(_t="$(date +'%s')" && printf "%s\n" "$((_t * _t))").LOG"
 
     _cleanup() {
         {
-            [ -n "${PARALLEL_UPLOAD}" ] && rm -f "${TMPFILE:?}"*
+            rm -f "${TMPFILE:?}"*
+
+            # first kill token service pid
+            kill "${ACCESS_TOKEN_SERVICE_PID}"
+
+            # grab and kill all chidren processes of the script
+            # https://askubuntu.com/a/512872
+            # shellcheck disable=SC2086
+            kill $(ps --ppid="$$" -o pid=)
```

Comment:
Codacy found an issue: [Quote this to prevent word splitting.](https://app.codacy.com/manual/labbots/google-drive-upload/pullRequest?prid=6363604)